src/euphorie/client/browser/risk.py

```diff
@@ -447,8 +451,43 @@ def proceed_to_next(self, reply):
         )
         return self.request.response.redirect(url)
 
+    @property
+    @memoize
+    def solutions(self):
+        if not self.risk:
+            return []
+        else:
+            return self.risk._solutions
+
     @memoize
     def get_existing_measures(self):
+        saved_standard_measures = {
+            getattr(measure, "solution_id", ""): measure
+            for measure in self.context.in_place_standard_measures
+        }
+        existing_measures = []
+        for solution in self.solutions:
+            existing_measures.append(
+                {
+                    "text": solution.action,
+                    "active": solution.id in saved_standard_measures,
+                    "solution_id": solution.id,
+                    "plan_type": "in_place_standard",
+                }
+            )
+        for measure in self.context.in_place_custom_measures:
+            existing_measures.append(
+                {
+                    "text": measure.action,
+                    "active": True,
+                    "solution_id": measure.id,
+                    "plan_type": "in_place_custom",
+                }
+            )
+        return existing_measures
+
+    @memoize
+    def get_existing_measures_old(self):
```

Comment:
I do not see this called anywhere:
```
[ale@emily Euphorie]$ grep -R get_existing_measures_old
src/euphorie/client/browser/risk.py:    def get_existing_measures_old(self):
src/euphorie/client/browser/risk.py:    def get_existing_measures_old(self):
```

Reply:
Yes, true. I kept it as a "backup", in case I needed to go back to fetching legacy data.
I will remove it once I have gotten feedback from several countries that they are happy with the result of the migration.